source/renderer/app/components/staking/stake-pools/StakePools.js

```diff
@@ -171,6 +178,20 @@ export default class StakePools extends Component<Props, State> {
       isLoading ? styles.isLoading : null,
     ]);
 
+    const smashServer = smashServerUrl
+      ? getSmashServerNameFromUrl(smashServerUrl)
+      : null;
+
+    const smashSettings = smashServer && (
+      <button onClick={onSmashSettingsClick} className={styles.smashSettings}>
+        <span>&nbsp;- {smashServer}</span>
```

Comment:
This could be like this
<span>{\` \`}- {smashServer}</span>
This is generated when using automatic prettier formatting after using simple whitespace

Reply:
Done